docs/release-notes/release-notes-0.13.4.md

```diff
@@ -16,15 +16,13 @@ with 'make go-build-noui' and 'make go-install-noui'](https://github.com/lightni
 
 ### LND
 
-### Loop
-
-### Pool
+* [Update the integrated LND version to
+  v0.18.3-beta] (https://github.com/lightninglabs/lightning-terminal/pull/843)
```

Comment:
still not rendering correctly. Need to remove the space between the [] and the ()

same below

Reply:
Thanks :pray:!